packages/wordpress/components/blockStyles/blocks/heading.js

```diff
@@ -1,7 +1,7 @@
 import { css } from 'styled-components';
 import { link } from './utils';
 
-/* eslint-disable indent */
+/* eslint-disable indent, import/prefer-default-export */
```

Comment:
I wonder if it's worth adding an `.eslint` file in `blockStyles` if this is a common rule(s) we are disabling.

```
{
    "rules": {
        "import/prefer-default-export": "off"
    }
}
```

Reply:
hmmmm true, that might be helpful. this is pretty common throughout the lede theme too...maybe we should just disable across the board within lede?

Reply:
Oh wait, I already disabled it in irving core but didn't remove these eslint disable lines.